app/models/person.rb

```diff
@@ -127,6 +131,21 @@ class Person < ApplicationRecord
       .where('given_name like :pattern OR family_name like :pattern OR display_name like :pattern
         OR emails.address like :pattern', pattern: pattern)
   }
+  scope :has_listings, ->(community) do
+    joins("INNER JOIN `listings` ON `listings`.`author_id` = `people`.`id` AND `listings`.`community_id` = #{community.id} AND `listings`.`deleted` = 0").distinct
+  end
+  scope :has_no_listings, ->(community) do
+    joins("LEFT OUTER JOIN `listings` ON `listings`.`author_id` = `people`.`id` AND `listings`.`community_id` = #{community.id} AND `listings`.`deleted` = 0")
+    .where(listings: {author_id: nil}).distinct
+  end
+  scope :has_stripe_account, -> { where(id: StripeAccount.active_users.select(:person_id)) }
```

Comment:
`stripe_accounts` table is missing index on `person_id`. Right now, the table is not large, so that's not an issue, but the index should be added.